recipes/database.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Cookbook Name:: timesync
+# Recipe:: database
+#
+# Copyright 2015 Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+bash 'database' do
+  user 'root'
+  cwd "#{node['timesync']['application_path']}/source"
+  code <<-EOH
+    npm run migrations
+    chown #{node['timesync']['user']}:#{node['timesync']['group']} dev.sqlite3
```

Comment:
Since this is temporary, I'm going to leave it as this. When we refactor to have a postgres backend, then I'll make it nowhere near as gross :)